scripts/utils.ts

```diff
@@ -383,7 +383,7 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
     return {
```

Comment:
There's another error message a couple lines above here, make the prefix "SSM Command Failed to send:"